templates/user/account_delete_form.html

```diff
@@ -0,0 +1,70 @@
+{% extends "base.html" %}
+
+{% block title %}My user profile{% endblock title %}
+
+{% block content %}
+<!-- Begin Page Content -->
+<div class="container-sm py-5">
+    <!-- Main Content Here -->
+    <div class="card border-0 shadow p-4">
+
+        <form action="{% url 'delete_account_post_handler' user.id %}" method="POST">
+            {% csrf_token %}
+            <div class="card-body">
+                <div class="row">
+                    <div class="col">
+                        <h5 class="card-title mb-0">Delete Account</h5>
+
+                        <div class="row pt-4">
+                            <p class="alert alert-warning" role="alert">
```

Comment:
Lets use danger here instead?
```suggestion
                            <p class="alert alert-danger" role="alert">
```